examples/flow.cpp

```diff
@@ -356,9 +356,9 @@ try
 
     // Solver for Newton iterations.
     std::unique_ptr<NewtonIterationBlackoilInterface> fis_solver;
-    if (param.getDefault("use_interleaved", false)) {
+    if (param.getDefault("use_interleaved", true)) {
         fis_solver.reset(new NewtonIterationBlackoilInterleaved(param, parallel_information));
-    } else if (param.getDefault("use_cpr", true)) {
+    } else if (param.getDefault("use_cpr", false)) {
```

Comment:
I disagree with this, if we are not using the interleaved solver we should default to the CPR one, not the simple one (which is useless for anything larger than SPE1).